script.ts

```diff
@@ -0,0 +1,12 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
```

Comment:
do we still need this now that we have the cli?

Reply:
Probably not